// angular2-drag-scroll/src/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const core_1 = require("@angular/core");
const drag_scroll_1 = require("./drag-scroll");
const drag_scroll_item_1 = require("./drag-scroll-item");
const index_1 = require("./device-info/index");
class DragScrollModule {
}
DragScrollModule.decorators = [
    { type: core_1.NgModule, args: [{
                imports: [
                    index_1.DeviceDetectorModule
                ],
                exports: [
                    drag_scroll_1.DragScroll,
                    drag_scroll_item_1.DragScrollItem
                ],
                declarations: [
                    drag_scroll_1.DragScroll,
                    drag_scroll_item_1.DragScrollItem
                ]
            },] },
];
DragScrollModule.ctorParameters = () => [];
exports.DragScrollModule = DragScrollModule;
function DragScrollModule_tsickle_Closure_declarations() {
    DragScrollModule.decorators;
    DragScrollModule.ctorParameters;
}
var drag_scroll_2 = require("./drag-scroll");
exports.DragScroll = drag_scroll_2.DragScroll;
var drag_scroll_item_2 = require("./drag-scroll-item");
exports.DragScrollItem = drag_scroll_item_2.DragScrollItem;
